### ISSUE TITLE
showing the service status "The requested address is not valid in this context" in Debian and other questions

### ISSUE THREAD

**Issue by NoCodeForOldMen** [hirschmann/nbfc#599]
Hello,

thank you for creating this program. I need some help, maybe somebody has an idea.

nbfc version: 1.6.1.0
OS: Debian testing
Build via github, no Docker
Notebook: Lenovo Thinkpad 13 Gen2

After installing I couldn't enable the service. I did #2:

> To install the NBFC service, copy nbfc/Linux/nbfc.service and nbfc/Linux/nbfc-sleep.service into /etc/systemd/system/

and I did 

> To enable and start the service, open a terminal and type `systemctl enable nbfc --now`

when I did `mono /opt/nbfc/nbfc.exe status --all` it said, that the service wasn't enabled.
I couldn't enable it. So I rebooted my notebook and after that I could once again choose the config:
`mono /opt/nbfc/nbfc.exe config --apply Lenovo Thinkpad 13 2191
`and the reply was 
> Extra positional arguments found: Thinkpad 13 2191

Which doesn't sound bad.

However when I type 
`mono /opt/nbfc/nbfc.exe status -a`
it says 
> The requested address is not valid in this context

and that goes for all the other commands like 
`mono /opt/nbfc/nbfc.exe start -e`
as well.
I haven't added the nbfc to my path because it looks messy and I do not know how to add it.
My path is 

> export PATH=/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/usr/share/games:/usr/local/sbin:/usr/sbin:/sbin:~/local/bin

I can only acces nbfc through mono. If I try to access it via nbfc or nbfc.exe I get command not found (which is due to the missing path I guess)

tl;dr: The installation was somewhat successful. I cannot enable the systemd service. I can choose my notebook profile but not use anything else in nbfc

**Comment by snoggelito** [comment]
Hi there 

> I did mono /opt/nbfc/nbfc.exe status --all it said, that the service wasn't enabled.
I couldn't enable it. So I rebooted my notebook and after that I could once again choose the config:
mono /opt/nbfc/nbfc.exe config --apply Lenovo Thinkpad 13 2191 and the reply was

    Extra positional arguments found: Thinkpad 13 2191

The problem here is that it's supposed to be mono /opt/nbfc/nbfc.exe config --apply **"Lenovo Thinkpad 13 2191"**
